release/__init__.py

```diff
@@ -5,3 +5,5 @@
 # The full license is in the file LICENSE.txt, distributed with this software.
 # -----------------------------------------------------------------------------
 from __future__ import annotations
+
+BOKEHJS_BUCKETS = [("cdn.bokeh.org", "us-east-1"), ("cdn-backup.bokeh.org", "us-west-2")]
```

Comment:
```suggestion
BOKEHJS_BUCKETS = (("cdn.bokeh.org", "us-east-1"), ("cdn-backup.bokeh.org", "us-west-2"))
```

Nit, but I think it is more correct to use an immutable tuple than a list.

Reply:
https://github.com/bokeh/bokeh/pull/14517/commits/96bdbcde049580da33146e282274264b926eed91